backend/lib/facade.js

```diff
@@ -0,0 +1,44 @@
+const mongoose = require('mongoose');
+
+class Facade {
+  constructor(name, schema) {
+    this.model = mongoose.model(name, schema);
+  }
+
+  create(body) {
+    const model = new this.model(body);
+    return model.save();
+  }
+
+  find(...args) {
+    return this.model
+      .find(...args)
```

Comment:
Is there a reason to clone `args` rather than just passing them through?

Reply:
one of those things that came this way after i scaffolded the project, but i'm guessing it's this way because it makes it more obvious that you can add additional logic if you want. again, just me being lazy. didn't think about the implementation too much yet other than what it does.